src/redux/constructor/constructor-basic/constructor-basic.sagas.js

```diff
@@ -0,0 +1,17 @@
+import { call, put, takeEvery } from 'redux-saga/effects';
+import { getConstructorBasicById } from './constructor-basic.operations';
+import { GET_CONSTRUCTOR_BASIC } from './constructor-basic.types';
+import { setConstructorBasic } from './constructor-basic.actions';
+
+export function* handleConstructorBasicLoad({ payload: id }) {
+  try {
+    const basic = yield call(getConstructorBasicById, id);
+    yield put(setConstructorBasic(basic));
+  } catch (error) {
+    console.error(error);
```

Comment:
In sagas we use another saga for error

![image](https://user-images.githubusercontent.com/59802190/106138694-22551000-6175-11eb-8750-ee19af80e67e.png)
![image](https://user-images.githubusercontent.com/59802190/106138744-34cf4980-6175-11eb-8e60-8ff911af4f66.png)

Reply:
fixed